### DISCUSSION TITLE
Multiple Authentication methods at the same time, meaning Ldap OR SSO possible?

### DISCUSSION THREAD

**Discussion by pparasuram** [netbox-community/netbox#10733] [General]
Hello,

Is it possible to have:

Multiple Authentication methods at the same time, meaning Ldap OR SSO possible?

That is user can click SSO link OR enter username and password for Azure SSO (for example).

Meaning both methods should be possible, NOT as a failover method.

Prakash

**Comment by jeremystretch** [comment]
No; only one remote authentication backend can be configured.
